config/database_test.go

```diff
@@ -436,26 +436,13 @@ func TestDatabaseStoreSave(t *testing.T) {
 		},
 	}
 
-	t.Run("set without save", func(t *testing.T) {
+	t.Run("set with automatic save", func(t *testing.T) {
```

Comment:
Let's remove `Test*StoreSave` altogether, and integrate this test case into `Test*StoreSet` instead.

Reply:
Done!